tests/unit/src/test/scala/tests/decorations/SyntheticDecorationsLspSuite.scala

```diff
@@ -852,4 +852,38 @@ class SyntheticDecorationsLspSuite extends BaseLspSuite("implicits") {
       )
     } yield ()
   }
+
+  test("value-of") {
+    for {
+      _ <- initialize(
+        """|/metals.json
+           |{
+           |  "a": {}
+           |}
+           |/a/src/main/scala/Main.scala
+           |object O {
+           |  def foo[Total <: Int](implicit total: ValueOf[Total]): Int = total.value
+           |  val m = foo[500]
+           |}
+           |""".stripMargin
+      )
+      _ <- server.didChangeConfiguration(
+        """{
+          |  "show-implicit-arguments": true
+          |}
+          |""".stripMargin
+      )
+      _ <- server.didOpen("a/src/main/scala/Main.scala")
+      _ <- server.didSave("a/src/main/scala/Main.scala")(identity)
+      _ = assertNoDiagnostics()
+      _ = assertNoDiff(
+        client.workspaceDecorations,
+        """|object O {
+           |  def foo[Total <: Int](implicit total: ValueOf[Total]): Int = total.value
+           |  val m = foo[500](new ValueOf())
```

Comment:
```suggestion
           |  val m = foo[500](new ValueOf(...))
```
maybe ? Since the real constructor would be `new ValueOf(500)` but that would be hard to do I think.

Reply:
The single argument that is there is `Empty`, so we could print it as `...` if that makes sense, but I don't really understand what is the meaning of `Empty`.

Reply:
I think it's an issue with semanticdb not being able to figure out or encode that actual argument.

Reply:
Does `Empty` always mean: "something is there, but I don't understand it"? Because if it does, it makes perfect sense to print it as `...` not just in this case but in general.

Reply:
I think so, not 100% sure, but I would print it here anyway